account_statement_sale_order/__openerp__.py

```diff
@@ -0,0 +1,37 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2013-14 Akretion
+#    @author Chafique DELLI <chafique.delli@akretion.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Bank Statement Sale Order',
+    'version': '0.1',
+    'category': 'Finance',
+    'license': 'AGPL-3',
+    'summary': '',
+    'description': """
+    The goal of this module is to create a self-completion method fill:
```

Comment:
Extract this to a README.rst file and remove from here. You should explain a bit more the purpose of the module (I didn't know very well yet), and use the OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst